docs/recipes/lighthouse-plugin-example/readme.md

```diff
@@ -7,8 +7,10 @@
  
 ## To run
 
-- Install as a (peer) dependency, parallel to `lighthouse`.
-- `npx lighthouse https://example.com --plugins=lighthouse-plugin-example --view`
+1. Install the plugin as a (peer) dependency, parallel to `lighthouse`.
+1. Install `lighthouse` v5 or later.
+2. Install the plugin as a (peer) dependency, parallel to `lighthouse`.
+3. Run `npx lighthouse lighthouse https://example.com --plugins=lighthouse-plugin-example --view`
```

Comment:
hm so trying the `npx -p lighthouse` method it seems to force install lighthouse everytime which is not what we want

on `npx@6.4.1` running the original `npx lighthouse https://example.com` seems to work just fine. @connorjclark are you able to still reproduce that bug you filed?

Reply:
> on `npx@6.4.1` running the original `npx lighthouse https://example.com` seems to work just fine. @connorjclark are you able to still reproduce that bug you filed?

hmm, on `npx@6.9.0` I am getting the problems. It does *sort of* run lighthouse, but if you include flags (some flags?) it messes things up.

@patrickhulce are you able to run `npx lighthouse https://example.com --plugins=lighthouse-plugin-not-a-real-plugin` and get a `Runtime error encountered: Unable to locate plugin` message?

Reply:
@brendankenny yup!

![image](https://user-images.githubusercontent.com/2301202/64715322-d8875f00-d485-11e9-8964-9b8992877af2.png)

Reply:
well I would say it's a bug introduced in a later version of npx but @connorjclark's original report was for 6.4.1 as well...

Reply:
![image](https://user-images.githubusercontent.com/4071474/64717218-b5a27e00-d478-11e9-81db-bf94597af0ec.png)
didnt work for me for 6.9.0. It instead runs the other `bin` script which just launches chrome.

Reply:
ditto for node 12 / npx 6.10.2

![image](https://user-images.githubusercontent.com/4071474/64717403-1e89f600-d479-11e9-8cb2-3fe16ce5471c.png)

Reply:
Make sure you are not in a folder with `lighthouse` installed as a dep. Otherwise, npx defers to `node_modules/.bin` for resolving.

Reply:
Even a globally installed lighthouse probably too^

Reply:
> Make sure you are not in a folder with lighthouse installed as a dep. Otherwise, npx defers to node_modules/.bin for resolving.

wait but isn't that the whole point of this guide on how to use plugins? we tell folks to install lighthouse in their project and right next to it their plugin?

Reply:
I wasn't reading the context of this PR just answering the npx question :) yeeea if we expect lighthouse to be installed in the folder then `npx lighthouse ...` is the way to go.

Unfortunate that there's this bifurcation in behavior ..

Reply:
what was the end result of this?

The `npx lighthouse lighthouse ...` call in the current commit doesn't work for me (it runs the same as the old `npx lighthouse`) but `npx -p lighthouse lighthouse...` *does* work. It seems like @connorjclark your advice to not check in a folder with lighthouse installed in it is to replicate the broken behavior...for the `-p` version it should work whether it's installed locally or not :)

I say we go with the `-p` since it works and at worst it just reinstalls lighthouse...but if you're really running a plugin (not just trying out an example plugin) you likely already have a way to run lighthouse (global install, local call within the repo, etc) and so won't be using npx.

Reply:
the `-p lighthouse lighthouse` format sgtm. afaict the downside is that it always bypasses the local project's defined package versions and installation, and links the package and its deps in a tmp folder for every run. I killed my connection and verified it at least uses npm's global cache.

Reply:
yeah I'm pretty sure `npx lighthouse lighthouse` is just a typo from copying my suggestion of `npx -p lighthouse lighthouse` so I don't think anyone wants that one :) 👍 

I really dislike reinstalling every time and `npx -p lighthouse lighthouse <url>` when run in a project that has lighthouse and lighthouse-plugin-foo installed *does not* recognize and use the locally available plugin on `6.4.1`. I'm assuming because it's doing some global reinstall and not using the local build. Since this is the context of the guide here, I would strongly prefer to not give out a known broken command. At this point I would just rather list both, `npx` does more harm than good 

```
./node_modules/.bin/lighthouse --plugins=...
```

or 

```
npm install -g lighthouse lighthouse-plugin-...
lighthouse
```

:( I wish it weren't broken but that just seems worse

Reply:
Apologies for not being responsive as I've been working on the actual development of my plugin. So is the preferred documentation here just to use

`./node_modules/.bin/lighthouse` --plugins=...